src/load/azext_load/data_plane/load_test/custom.py

```diff
@@ -70,6 +71,7 @@ def create_test(
             key_vault_reference_identity=key_vault_reference_identity,
             subnet_id=subnet_id,
             split_csv=split_csv,
+            disable_public_ip=disable_public_ip,
```

Comment:
Should we throw error if `disable_public_ip` is sent for a public load test similar to our existing service side validations?

Reply:
Service side validation are good enough with a clear code and message incase of invalid scenarios.